File: Design_Hashset/hashset.py
class MyHashSet:

    def __init__(self):
        self.payload = [False] * 1000001

    def add(self, key: int) -> None:
        self.payload[key] = True

    def remove(self, key: int) -> None:
        self.payload[key] = False

    def contains(self, key: int) -> bool:
        return self.payload[key]


# Your MyHashSet object will be instantiated and called as such:
# obj = MyHashSet()
# obj.add(key)
# obj.remove(key)
# param_3 = obj.contains(key)

File: Design_Hashset/hashset.go
package designhashset

type MyHashSet struct {
	payload [1_000_001]bool
}

func Constructor() MyHashSet {
	return MyHashSet{payload: [1_000_001]bool{}}
}

func (h *MyHashSet) Add(key int) {
	h.payload[key] = true
}

func (h *MyHashSet) Remove(key int) {
	h.payload[key] = false
}

func (h *MyHashSet) Contains(key int) bool {
	return h.payload[key]
}
